CHANGELOG.md

```diff
@@ -5,10 +5,14 @@
 - Breaking: `.git/` is now ignored by default when using `--hidden` / `-H`, use `--no-ignore` / `-I` or
   `--no-ignore-vcs` to override, see #1387 and #1396 (@skoriop)
 
+
 ## Bugfixes
 
 ## Changes
 
+- The default number of threads is now constrained to be at most 16. This should improve startup time on
+  systems with many CPU cores. (#1203)
```

Comment:
Maybe I'm missing something, but isn't `MAX_DEFAULT_THREADS` actually 20?

I think it would also be good to note that maximum in the CLI help.

Reply:
Ugh. I had a local change to set it to 16 instead of 20, but that didn't get included in the commit.

Reply:
We may want to tweak the limit again post #1422 anyway.  I get better perf all the way up to 48 threads on my 24 core CPU and the startup overhead is minimal now

Reply:
Actually I just checked more carefully and the ideal thread count for me is 24 on my 24 core/48 thread CPU.  I'm curious how this scales on different machines.

```
  ./fd-batch -u -j24 --search-path ~/code/bfs/bench/corpus/chromium ran
    1.02 ± 0.04 times faster than ./fd-batch -u -j28 --search-path ~/code/bfs/bench/corpus/chromium
    1.02 ± 0.03 times faster than ./fd-batch -u -j32 --search-path ~/code/bfs/bench/corpus/chromium
    1.07 ± 0.03 times faster than ./fd-batch -u -j20 --search-path ~/code/bfs/bench/corpus/chromium
    1.07 ± 0.03 times faster than ./fd-batch -u -j36 --search-path ~/code/bfs/bench/corpus/chromium
    1.08 ± 0.03 times faster than ./fd-batch -u -j40 --search-path ~/code/bfs/bench/corpus/chromium
    1.09 ± 0.02 times faster than ./fd-batch -u -j48 --search-path ~/code/bfs/bench/corpus/chromium
    1.09 ± 0.02 times faster than ./fd-batch -u -j44 --search-path ~/code/bfs/bench/corpus/chromium
    1.21 ± 0.02 times faster than ./fd-batch -u -j16 --search-path ~/code/bfs/bench/corpus/chromium
    1.52 ± 0.03 times faster than ./fd-batch -u -j12 --search-path ~/code/bfs/bench/corpus/chromium
    2.15 ± 0.04 times faster than ./fd-batch -u -j8 --search-path ~/code/bfs/bench/corpus/chromium
```